THEMES/EdgeTXClean/theme.yml

```
@@ -0,0 +1,19 @@
+---
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)